src/main/java/Duke.java

```diff
@@ -1,10 +1,103 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static int addTaskItem(Task[] TaskList, String item, int numOfItems, String typeOfTask){
+
+        switch(typeOfTask){
+        case "todo":
+            TaskList[numOfItems] = new Todo(item,numOfItems);
+            break;
+
+        case "deadline":
+            if(item.contains("/by")) {
+                String dateOfDeadline = item.substring(item.indexOf("/by"));
+                TaskList[numOfItems] = new Deadline(item.substring(0, item.indexOf("/by") - 1), dateOfDeadline, numOfItems);
+            }else{
+                TaskList[numOfItems] = new Deadline(item,numOfItems);
+            }
+            break;
+
+        case "event":
+            if(item.contains("/at")) {
+                String dateOfEvent = item.substring(item.indexOf("/at"));
+                TaskList[numOfItems] = new Event(item.substring(0, item.indexOf("/at") - 1), dateOfEvent, numOfItems);
+            }else{
```

Comment:
Your code looks clean but I would suggest putting spaces like "} else {" to improve readability. but otherwise well done! :)

Reply:
I would say "do it" rather than suggest!